homeassistant/components/myuplink/entity.py

```diff
@@ -0,0 +1,29 @@
+"""Provide a common entity class for myUplink entities."""
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import MyUplinkDataCoordinator
+
+
+class MyUplinkEntity(CoordinatorEntity[MyUplinkDataCoordinator], Entity):
```

Comment:
Noticed this clean up, when browsing the code.
```suggestion
class MyUplinkEntity(CoordinatorEntity[MyUplinkDataCoordinator]):
```